gmcs/linglib/information_structure.py

```diff
@@ -542,7 +542,7 @@ def customize_information_structure_pos_once(mylang, ch, rules, infostr_type, in
                 #         [ SYNSEM.R-PERIPH +,
                 #           HEAD-DTR.SYNSEM [ R-PERIPH +,
                 #                             LOCAL.CONT.HOOK.ICONS-KEY $,
-                #                             NON-LOCAL.SLASH 0-dlist ] ]."""
+                #                             NON-LOCAL.SLASH 0-alist ] ]."""
                 #ph_types['subj-head-phrase'] = tdl.replace('$', infostr)
                 #ph_rules['subj-head-phrase'] = 'subj-head'
```

Comment:
I suppose!!

Reply:
Actually, no, I don't think I want to remove Sanghoun's commented out code just yet because there are places in his library which are unfinished which I think may guide further development. So I'd leave it.

Reply:
Then I think these blocks should have some comment describing what we want to do with them. E.g., do they pertain to some specific part of his thesis? Are they related to some matrix Bug? But this is all outside the scope of this PR.

Reply:
Agreed, but this is unrelated to the PR :). This comment has always been there, I just accidentally replaced the diff-list there with an append-list :). I could just revert the change :).

Reply:
I went ahead and reverted this commented out part to diff-lists, perhaps there will be less confusion, more obvious that that's old code that is there for information purposes. As for adding more comments explaining what the code is for, that's separate (we can create an issue...)

Reply:
I think it's fine as-is, except for any sort of descriptive comment. If the Matrix is moving to append-lists, then the features this commented-out code supposedly implements would need to use append-lists, too.

Reply:
There's lots of places like this throughout the customization system (and matrix.tdl!) Perhaps we should just be consistent (which means not touch them in this PR). I will create an issue.

Reply:
Ok that's fine.